server/src/useCases/report/reportSave/handleSaveRequest.ts

```diff
@@ -175,10 +178,10 @@ module.exports = class ReportUploadRequestHandler {
     this.saveToDatabase(repository, report);
   }
 
-  saveToDatabase(repository, data) {
+  saveToDatabase(repository: any, data: any): void {
```

Comment:
Can you add specific types for repository and data?

Reply:
done